src/ndv/models/_roi_model.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pydantic import field_validator
+
+from ndv.models._base_model import NDVModel
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+class RectangularROIModel(NDVModel):
+    """Representation of an axis-aligned rectangular Region of Interest (ROI).
+
+    Parameters
+    ----------
+    visible : bool
+        Whether to display this roi.
+    bounding_box: tuple[Sequence[float], Sequence[float]]
+        The minimum point and the maximum point contained within the region.
+        Using these two points, an axis-aligned bounding box can be constructed.
+    """
+
+    visible: bool = True
+    bounding_box: tuple[tuple[float, float], tuple[float, float]] = ((0, 0), (0, 0))
+
+    @field_validator("bounding_box")
```

Comment:
```suggestion
    @field_validator("bounding_box", mode='after')
```

mode after is the default, but i like to be explicit because it's an important distinction.  In this case, it means that you can assume that the `bb` argument being passed in is already going to be a valid tuple and you don't need to check again.